### PR TITLE
add plaster 

### PR DESCRIPTION
**Pull request by bkille**
Add the [plaster](https://gitlab.com/treangenlab/plaster) tool

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by bkille** [comment]
@BiocondaBot please add label

File: recipes/plaster/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+about:
+  home: https://gitlab.com/treangenlab/plaster 
+  license: MIT
+  license_file: LICENSE
+  summary: Plaster is a program for quick linear pangenome construction.
+
+extra:
+  skip-lints:
+    - folder_and_package_name_must_match
```
Comment (bgruening) [inline recipes/plaster/meta.yaml:37]: why not also rename the folder, I guess this makes it easier to find the recipe for developers

**Comment by bgruening** [comment]
Thanks @bkille for your contribution!

File: recipes/plaster/meta.yaml (hunk 1 above)
Comment (bkille) [reply recipes/plaster/meta.yaml:37]: I tried by adding the `folder:` section to `source`, but then my local build started failing for another reason... ): I'm sure there's a correct way to do it, but I'm still new to figuring out these `meta.yml`

Comment (bgruening) [reply recipes/plaster/meta.yaml:37]: the bioconda folder is meant, you should rename the recipes/plaster folder :)

Comment (bkille) [reply recipes/plaster/meta.yaml]: Ahh understood. I have made the change. Thanks!

**Review by innovate-invent — CHANGES_REQUESTED** [review]

File: recipes/pan-plaster/meta.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ test:
     - example/
     - example/*
   commands:
-    - plaster $PREFIX/bin/example/*.fasta
```
Comment (innovate-invent) [inline recipes/pan-plaster/meta.yaml]: Keep the more elaborate test, but just reference the source files for the example data.
You might also want to validate the output data using grep.

**Review by innovate-invent — CHANGES_REQUESTED** [review]

File: recipes/pan-plaster/meta.yaml (hunk 2 above)
Comment (bkille) [reply recipes/pan-plaster/meta.yaml]: I've moved the more elaborate test to `run_test.sh`

File: recipes/pan-plaster/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+about:
+  home: https://gitlab.com/treangenlab/plaster 
+  license: MIT
+  license_file: LICENSE
+  summary: Plaster is a program for quick linear pangenome construction.
+
+extra:
+  skip-lints:
+    - folder_and_package_name_must_match
```
Comment (innovate-invent) [inline recipes/pan-plaster/meta.yaml:37]: I think this needs to be removed now that the folder has been renamed

**Review by innovate-invent — CHANGES_REQUESTED** [review]

**Review by innovate-invent — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

**Comment by bkille** [comment]
@BiocondaBot please merge

**Pull request merged**